docs/sphinx/source/clearsky.rst

```diff
@@ -327,9 +374,15 @@ from the `ECMWF <https://software.ecmwf.int/wiki/display/WEBAPI/Access+ECMWF+Pub
 and `SoDa <http://www.soda-pro.com/web-services/radiation/cams-mcclear>`_.
 
 Aerosol optical depth is a function of wavelength, and the Simplified
-Solis model requires AOD at 700 nm. Models exist to convert AOD between
-different wavelengths, as well as convert Linke turbidity to AOD and PW
-[Ine08con]_, [Ine16]_.
+Solis model requires AOD at 700 nm. The function,
```

Comment:
delete ``The function`` and commas. I think the formatting makes it clear that it's a function. But you can keep it if you prefer this way.